index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>IsaMarvin | Software Engineer</title>
+  </head>
+  <body>
+    <header class="header">
+      <img src="./images/my-logo.svg" alt="logo" />
+      <img
+        src="./images/hamberger.svg"
+        alt="hamburger"
+      />
+    </header>
```

Comment:
- Good job wrapping this section in a `header` tag, but to help with it semantics of your page, please wrap your logo and hamburger icon in a `nav` tag 🙏